### DISCUSSION TITLE
Blazor WebAssembly Google authentication | not abble to logout

### DISCUSSION THREAD

**Discussion by BernhardNinaus** [dotnet/aspnetcore#23878] [General] [answered]
Hey.

I followed another [Discussion](https://github.com/dotnet/aspnetcore/issues/21326) for the authentication to get it working. I also tired out the solution from @javiercn ([Repo](https://github.com/javiercn/BlazorGoogleAuthSample)).

My problem is, that I cannot successfull logout on the client and login to another google account.
When i click "Log out" (top right corner) then I get redirected to https://localhost:5000/authentication/logged-out with the message "You are logged out.". 
The JS log:
![Capture](https://user-images.githubusercontent.com/58560010/87245417-6553ce00-c445-11ea-9088-f0afb148b7d1.PNG)

When i reload the site, I immediately get back logged in.

Then I tried out the iPS from https://identityserver.io/ with changing only these parameters:
```
    options.ProviderOptions.Authority = "https://demo.identityserver.io/";
    options.ProviderOptions.ClientId = "interactive.public";
    options.ProviderOptions.ResponseType= "code";
```
ResponseType = "code" because there is no other ClientId setup for implicit flow and also there is no client secret? 
But everything works and I do really get logged out and can chose to login as another user.

Am I missing something in the spec in general? 
If i would have used "code" and Google then I would get an error for missing secret, which is okay and expected.
With identityserver, i can ask for scopes and access token and can call the api.

Thank you for your help, I really appreciate it.

**Comment by javiercn** [comment, marked as answer]
@BernhardNinaus thanks for contacting us.

Google doesn't support logout, so the best we can do is log you out locally.
